guides/doc-Planning_Guide/topics/Glossary.adoc

```diff
@@ -130,7 +130,9 @@ In relationship to a host, erratum is *applicable* if it updates a package insta
 [[varl-Glossary_of_Terms-External_Node_Classifier]]
 *External Node Classifier*:: A Puppet construct that provides additional data for a Puppet server to use when configuring hosts.
 {ProjectName} acts as an External Node Classifier to Puppet servers in a {Project} deployment.
-
++
+Note that the Puppet External Node Classifier is deprecated and will be removed in the next {Project} version.
```

Comment:
This is a bit more nuanced. In upstream it'll be moved to [foreman_puppet](https://github.com/theforeman/foreman_puppet) (currently undecided if it'll be included by default or not, also undecided how we'll handle it in upgrades: https://github.com/theforeman/foreman-installer/pull/695).

In downstream the foreman_puppet won't be shipped so there it will be removed.

To make it even more confusing, https://github.com/theforeman/foreman/pull/8510 (as I understand it now) will remove `/node/:certname` but not `/api/hosts/:id/enc` which is the same data.

The crucial part is that the environment and classes are removed from the ENC.

Reply:
This is also removed rather than deprecated.

Reply:
Changed as per comment.